src/core/midiMapConf.cpp

```diff
@@ -116,7 +116,7 @@ void parse_(Message& message)
 	message.value = strtoul(output.c_str(), nullptr, 16);
 
 	u::log::print("[parse] parsed chan=%d valueStr=%s value=%#x, offset=%d\n",
-			message.channel, message.valueStr.c_str(), message.value, message.offset);
+			message.channel, message.valueStr, message.value, message.offset);
```

Comment:
My question about the current tab-based coding style is how to align `message` with the `"`?

Reply:
I guess this assumes some very specific tab size.

Reply:
I think I use 8-char tabs in my files, and it seems to work well when browsing Giada code in overall.